src/Kernel.php

```diff
@@ -45,7 +45,9 @@ public function registerBundles()
 
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
-        $confDir = dirname(__DIR__).'/config';
+        $container->setParameter('container.autowiring.strict_mode', true);
+        $container->setParameter('container.dumper.inline_class_loader', true);
```

Comment:
I think both behavior are enabled by default in 4.0 so they are not necessary.

Reply:
Hmmm, not `container.dumper.inline_class_loader` IIRC.
For `container.autowiring.strict_mode`, it just has no effect, as the BC layer for the old autowiring system is removed.